src/html.js

```diff
@@ -183,7 +193,7 @@ function generateAttributes(params, { tag, log, _idRegistry }) {
 		// regular attributes
 		default:
 			// cf. https://html.spec.whatwg.org/multipage/syntax.html#attributes-2
-			if(/ |"|'|>|'|\/|=/.test(name)) {
+			if(/[ "'>/=]/.test(name)) {
```

Comment:
Good catch there; this should have been a character set from the start. 👍

Reply:
Thanks ;)